index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>
+    <section></section>
+    <nav class="navbar">
+        <div class="container">
+            <div class="logo">John doe </div>
+        <div class="menu"><img src="img/Union.svg" alt=""></div>
+    </div>
+     </nav>
+    <header class="header">
+        <div class="container">
+                    <div class="textInfo">
+             <h1>Hey There. I'm alex</h1>
```

Comment:
- [ ] Nice work @Aminocodex, but ...as per the requirement, please use a `p` element tag for this text on line 22